### DISCUSSION TITLE
Change colors on already created message box?

### DISCUSSION THREAD

**Discussion by nugward** [Akascape/CTkMessagebox#45] [Q&A]
Is it possible to change the colors of an already created message box? I have a dark mode toggle in my program and want the message box to match the current theme. I'm currently trying this approach, with a "dark mode check", but it does not work:

```
    msg = CTkMessagebox(
        title='Save?',
        message='Do you want to save before exiting?',
        icon='question',
        option_1='Cancel',
        option_2='No',
        option_3='Yes')

    if isDarkMode:
        msg.configure(button_color='#D2D2D2', button_hover_color='#7FC1CC', button_text_color='#000000')
    else:
        msg.configure(button_color='#383838', button_hover_color='#0C5653', button_text_color='#E9E9E9')
```

Am I doing this wrong? Or should I simply have create two different 'msg', one with light colors and one with dark colors, and displaying either one depending on if dark mode is enabled or not?

**Comment by Akascape** [comment]
configure method is not implemented, better spawn a new messagebox with the following parameters.
